client/homescreen/activity-panel/style.scss

```diff
@@ -1,67 +1,61 @@
-@mixin accordion-header {
-	.woocommerce-accordion-header {
-		order: 1;
-		text-align: left;
-		.woocommerce-accordion-title {
-			margin-right: $gap;
-		}
-	}
-}
-.woocommerce-accordion-card {
-	.components-card__header,
-	.components-panel__body-title .components-button {
-		font-size: 20px;
-		line-height: 28px;
-		color: $gray-900;
-		&:focus {
-			box-shadow: unset;
-			outline: unset;
+.woocommerce-activity-panel {
+	background: transparent;
+	border: 0;
+
+	.components-panel__body {
+		background: #fff;
+		border: 1px solid $gray-200;
+
+		&.is-opened {
+			margin-bottom: $gap-large;
+
+			.components-panel__body-toggle {
+				border-bottom: 1px solid $gray-200;
+			}
 		}
 	}
 
-	.components-card__header {
-		height: 60px;
-		@include accordion-header();
+	&.components-panel > .components-panel__body:last-child {
+		border-bottom: 1px solid $gray-200;
 	}
 
-	.components-panel__body-title {
-		margin: 0;
-		padding: $gap-small 0;
-		height: 60px;
-		.components-button {
+	.components-panel__row {
+		margin: 0 -#{$gap} -#{$gap} -#{$gap};
+		> div {
 			width: 100%;
-			padding-left: 24px;
-			font-style: normal;
-			font-weight: 400;
-			display: flex;
-			flex-direction: row;
+		}
+	}
 
-			@include accordion-header();
+	.components-panel__body-toggle {
+		border-radius: 0;
 
-			.woocommerce-accordion-header {
-				width: 50%;
-			}
-
-			& > span {
-				order: 2;
-				width: 50%;
-				text-align: right;
-				.components-panel__arrow {
-					width: 30px;
-					height: 30px;
-					vertical-align: text-top;
-				}
-			}
+		&:disabled {
+			opacity: 1;
 		}
 	}
 
+	.woocommerce-activity-panel {
+		margin-bottom: $gap-large;
+	}
+
+	.components-panel__body-title p {
+		margin-right: $gap;
+	}
+
 	.woocommerce-activity-card {
 		padding: $gap-largest/2 $gutter $gutter;
+		border: 0;
```

Comment:
I think we need to replace `border: 0` with `border-bottom: 1px solid $gray-200;` here to add the borders back between the rows.

Reply:
Gotcha.  I had removed this because I was seeing double border thickness in some areas.  Added this in with a small tweak:

```scss
		&:not(:last-of-type) {
			border-bottom: 1px solid $gray-200;
		}
```

Borders between activity cards, but not on the last item to avoid thickened borders.

<img width="697" alt="Screen Shot 2021-01-06 at 2 51 15 PM" src="https://user-images.githubusercontent.com/10561050/103813937-dd830100-502e-11eb-98bf-6f2ef4bbf661.png">

Reply:
Nice one👍